### ISSUE TITLE
(Priority 1) Vertically Flipped Sagittal & Coronal Orientation

### ISSUE THREAD

**Issue by spycolyf** [KitwareMedical/SlicerQReads#45]
Coronals are Vertically Flipped but superior orientation still pointing up for these images. 
Drop Box location: https://www.dropbox.com/s/tun2gklwoiqkbrs/BadOrientation.zip?dl=0

**Comment by jcfr** [comment]
Comment from Steve copied from https://discourse.slicer.org/t/slicerqr-development/15954/68:

> I had a quick look and that is DICOM Secondary Capture format (screen dump) so geometry is often not well preserved.
>
> In this case it loads correctly via the DICOM module but incorrectly with the regular Add Data. We always recommend the 
> DICOM module for data that is known to be in DICOM format.

**Comment by spycolyf** [comment]
This code loads all DICOM objects into the scene from a file folder. All the registered plugins are evaluated and the one with the highest confidence will be used to load the data. Files are imported into a temporary DICOM database, so the current Slicer DICOM database is not impacted.

```python
dicomDataDir = "c:/my/folder/with/dicom-files"  # input folder with DICOM files
loadedNodeIDs = []  # this list will contain the list of all loaded node IDs
  
from DICOMLib import DICOMUtils
with DICOMUtils.TemporaryDICOMDatabase() as db:
  DICOMUtils.importDicom(dicomDataDir, db)
  patientUIDs = db.patients()
  for patientUID in patientUIDs:
    loadedNodeIDs.extend(DICOMUtils.loadPatientByUID(patientUID))
```

**Comment by jcfr** [comment]
For reference snippet above was copied from https://www.slicer.org/wiki/Documentation/Nightly/ScriptRepository#How_to_load_DICOM_files_into_the_scene_from_a_folder

**Comment by spycolyf** [comment]
Hello, jc I’m off today and will be checking out your changes tomorrow. Will you be implementing any more of the priorities I listed in GitHub tonight? I’m planing on downloading and building late tonight. The next change that would have the most positive political impact is the image windows background colors issue. Backgrounds = black, reference line colors = #990000, #009900, and #000099. Scroll bar are fine as is.

The chair of the Applications Oversight Committee will be really happy about that. That was the thing he was most unhappy about and called me out at the meeting saying colors like that are for children. So he will be most happy with that change.

Thanks.

On Mar 8, 2021, at 7:19 PM, Jean-Christophe Fillion-Robin <notifications@github.com> wrote:

﻿

Closed #45<https://github.com/KitwareMedical/SlicerQReads/issues/45> via #51<https://github.com/KitwareMedical/SlicerQReads/pull/51>.

**Comment by jcfr** [comment]
| Without Slicer DICOM Modules | With Slicer DICOM Modules |
|---|---|
| ![image](https://user-images.githubusercontent.com/219043/110549558-3d198d80-8100-11eb-961c-e7ff739625de.png) |![image](https://user-images.githubusercontent.com/219043/110549429-0b082b80-8100-11eb-9773-6bfb1bf281f5.png)|


* Without Slicer DICOM Modules:

```
./SlicerQReads --python-code "folder='/home/jcfr/Projects/SlicerQReads-TestImages/BadOrientation'; import os; slicer.util.loadVolume(folder + '/' + os.listdir(folder)[0], {'singleFile': False})"
```

* With Slicer DICOM Modules

```
./SlicerQReads --python-code "from QReads import QReadsLogic; QReadsLogic.loadDICOMDataDirectory('/home/jcfr/Projects/SlicerQReads-TestImages/BadOrientation')"
```
